appveyor.yml

```diff
@@ -7,14 +7,22 @@ environment:
   global:
     DOWNLOAD_DIR: _download_cache
     APPVEYOR_SAVE_CACHE_ON_ERROR: true
-    PATH: C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Program Files\Git\cmd
+    PATH: C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Program Files\Git\cmd;C:\Program Files\7-Zip
   matrix:
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
```

Comment:
This should be named 2015?

Reply:
The "Visual Studio 2017" image has both VS2015 and VS2017 installed, see here for details: https://www.appveyor.com/docs/build-environment/#image-updates

The only upside from 2015 is afaics that it uses an older Windows and would fail on unintended Window 10 only API usage.